<launch>
    <arg name="mppi_param_path" default="$(find mppi_controller)/config/mppi_controller.yaml"/>
    <arg name="is_simulation" default="false"/>
    <arg name="is_localize_less_mode" default="false"/>

    <node pkg="mppi_controller" type="mppi_controller_node" name="mppi_controller" output="screen">
        <rosparam command="load" file="$(arg mppi_param_path)"/>
        <param name="is_simulation" value="$(arg is_simulation)"/>
        <param name="is_localize_less_mode" value="$(arg is_localize_less_mode)"/>
    </node>
</launch>